darwin/client.py

```diff
@@ -124,15 +123,15 @@ def get(self, endpoint: str, retry: bool = True, raw: bool = False):
             return response
         return response.json()
 
-    def put(self, endpoint: str, payload: Dict, retry: bool = True):
+    def put(self, endpoint: str, payload: Dict, retry: bool = False):
```

Comment:
Same here for `retry` (basically, in all places where it was `True` I liked `True`)